src/server.js

```diff
@@ -17,10 +18,21 @@ try {
     access_token: process.env.ACCESS_TOKEN,
     access_token_secret: process.env.ACCESS_TOKEN_SECRET,
   });
+  stream = twitBot.stream('statuses/filter', { track: 'Want to read more or fix an error? Visit Nkọwa okwu:' });
```

Comment:
Since the `track` text is referring to a string that could possibly change in the future, we should probably create a constant that `track` and random word tweet status message use. That way if we ever want to change the text in our status, `track` will automatically match with the updated text for free.

Reply:
Makes sense!